docs/en/open-source.rst

```diff
@@ -0,0 +1,53 @@
+.. include:: ../common/common_definitions.rst
+
+.. _open-source.rst:
+
+Open Source Releases for IT-Wallet Solutions  
+============================================
+
+In compliance with the LLGG Wallet mandates, IT-Wallet Solution Providers, both public and private, are required to make their source code available. This requirement emphasizes the importance of using open-source software within public administrations to promote transparency and reduce costs. This requirement is supported by the Italian Digital Administration Code (CAD) articles 68 and 69, European regulations (``Artificial Intelligence Act (AI Act)`` <https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A32024R1689>_, ``Cyber Resilience Act (CRA)`` <https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=OJ:L_202402847>_, ``Interoperable Europe Act`` <https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:52022PC0720>_), and best practices in open-source software development. This document defines the specific requirements for different stakeholders involved in the IT-Wallet ecosystem, including Wallet Providers, Credential Issuers, and Relying Parties.
+
+All the implementers, be these Wallet Providers, Credential Issuers or Relying Parties, owning the product code (from now on Open Source Project Owner) SHOULD follow industry best practices for open-source software, including proper documentation, version control, and community engagement. In particular:
+
+    - **Transparency and Documentation**: Open Source Project Owners SHOULD produce clear documentation and contribution guidelines.
+    - **Community Engagement**: Open Source Project Owners SHOULD foster active community involvement for development and support.
+    - **Version Control**: Open Source Project Owners MAY use systems like Git for managing code changes.
+    - **Security Practices**: Open Source Project Owners MUST produce regular code audits and secure coding standards.
+    - **Licensing**: Open Source Project Owners MUST use appropriate open-source licenses (e.g., EUPL, MIT, Apache 2.0, GPL, AGPL).
```

Comment:
Maybe we can add something like this, to exclude esoteric licenses like "GPL + COMMON CLAUSE":

> the license MUST be recognized as "free license" or "open source license" by the Free Software Foundation or the Open Source Initiative, without any added restrictions.

Reply:
https://interoperable-europe.ec.europa.eu/collection/eupl/solution/licensing-assistant/find-and-compare-software-licenses

Actually EUPL (1.2) seems to be 1st choice for IEA